Assets/Plugins/Source/Editor/EditorWindows/EOSSettingsWindow.cs

```diff
@@ -197,7 +195,14 @@ protected override void RenderWindow()
 
             if (_platformTabs != null && _platformConfigEditors.Count != 0)
             {
-                _selectedTab = GUILayout.Toolbar(_selectedTab, _platformTabs, TAB_STYLE);
+                var newlySelectedTabIndex = GUILayout.Toolbar(_selectedTab, _platformTabs, TAB_STYLE);
+
+                if (newlySelectedTabIndex != _selectedTab)
+                {
+                    _selectedTab = newlySelectedTabIndex;
+                    Debug.Log($"Selected config tab changed to {_selectedTab}.");
```

Comment:
Do we want to print this debug log every time a tab is selected? Currently, navigating around prints this out.

Reply:
No, we do not. I've removed it now. Marking this conversation as resolved and merging the PR.